ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -120,3 +120,56 @@ SENSOR_FDPM:
     - local_air_temperature_sensor
     - battery_charge_percentage_sensor
     - signal_strength_percentage_sensor
+
+SENSOR_ZA_CO_CO2:
+  description: "Atmosphere CO and CO2 combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZA
+  uses:
+  - thirtysecondrolling_average_zone_air_co_concentration_sensor
+  - zone_air_co_concentration_sensor
+  - zone_air_co2_concentration_sensor
+
+SENSOR_ZA_HCHO:
+  description: "Atmosphere formaldehyde (H-CHO) combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZA
+  uses:
+  - zone_air_formaldehyde_concentration_sensor
+
+SENSOR_ZA_OZONE_NO2:
+  description: "Atmosphere ozone and NO2 combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZA
+  uses:
+  - zone_air_no2_ozone_concentration_sensor_1
```

Comment:
Thought: maybe we make the internal sub-sensors optional?  That might make this type more reusable
zone_air_no2_ozone_concentration_sensor_1
zone_air_no2_ozone_concentration_sensor_2
zone_air_no2_probability_percentage_sensor

Reply:
accepted